src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -993,3 +995,15 @@ export const handleGetMetadata = [
   }),
   getMetadata,
 ] as ControllerHandler[]
+
+export const handleGetS3PresignedPostData: ControllerHandler<
+  unknown,
+  Record<string, S3.PresignedPost> | ErrorDto,
+  Record<string, number>
+> = async (req, res) => {
+  return getQuarantinePresignedPostData(req.body)
+    .map((presignedUrls) => res.json(presignedUrls))
+    .mapErr(() =>
+      res.status(StatusCodes.BAD_REQUEST).send({ message: 'error' }),
```

Comment:
suggestion: maybe a more descriptive error message would help 😅

Reply:
Good catch! Sorry this was a placeholder that I forgot about. Fixed in [441be21](https://github.com/opengovsg/FormSG/pull/6685/commits/441be21893d1426f47665b580349b46e777ab062).